setup/openchange_provision

```diff
@@ -97,5 +98,7 @@ if not opts.openchangedb:
     else:
         parser.print_help()
         sys.exit(1)
+elif opts.check:
```

Comment:
what if both opts.check and opts.openchangedb are specified?

it would be nice to give users an error message in that case, otherwise the behaviour might be unexpected (especially considering the description doesn't say that --check is mutually exclusive with --openchangedb).